### PR TITLE
Tuning

### PR DESCRIPTION
**Pull request by miraclestyle**
[Fine-tune data generator](https://trello.com/c/IowIWNoc/21-fine-tune-data-generator)

### CODE REVIEW

File: server/config/postgresql.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  user: 'elvinkosova',
+  host: 'localhost',
+  database: 'elvinkosova',
+  //password: '',
+  port: 5432,
+};
```
Comment (matt-wigg) [inline server/config/postgresql.js:7]: I wonder if you want to hide this information (gitignore) ?

File: server/database/generate/utils.js [hunk 2]
```diff
@@ -23,195 +23,32 @@ const checkOutHelper = () => {
...
-//     street_address: streetAddress,
-//     country_code: countryCode,
-//     editorial_text_one: editorialOne,
-//     editorial_text_two: editorialTwo,
-//     description_text_one: descriptionOne,
-//     description_text_two: descriptionTwo,
-//     description_text_three: descriptionThree,
-//   };
-// };
+const getSentences = (count, size) => {
```
Comment (matt-wigg) [inline server/database/generate/utils.js:34]: Awesome variable descriptions 🏅

**Review by matt-wigg — APPROVED** [review]
Nice one - looks good! 🚀

File: server/config/postgresql.js (hunk 1 above)
Comment (miraclestyle) [reply server/config/postgresql.js:7]: I decided to use it as a template for production. I don't expect the structure of it to change in the future.

**Pull request merged**